### PR TITLE
Add story feature let everyone know what's going on

### PR DESCRIPTION
**Pull request by timchengx**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,6 @@
 *.pyc
+.vscode/
+bin/
+include/
+lib/
+.Python
```
Comment (legnaleurc) [inline .gitignore]: Missing trailing newline.

File: bot/__main__.py [hunk 2]
```diff
@@ -137,19 +137,67 @@ def __init__(self):
...
 
 @command_filter(r'^/help(@\S+)?$')
 async def help(message, *args, **kwargs):
     return '\n'.join((
         '',
         '/ypc',
         '/ypc add <sentence>',
+        '/ypc addstory <id> <sentence>',
         '/ypc remove <id>',
+        '/ypc removestory <id> <sentence>',
```
Comment (legnaleurc) [inline bot/__main__.py:206]: Missing `/ypc story`.

File: bot/__main__.py [hunk 3]
```diff
@@ -137,19 +137,67 @@ def __init__(self):
             '',
             '/ypc',
             '/ypc add <sentence>',
+            '/ypc addstory <id> <sentence>',
             '/ypc remove <id>',
+            '/ypc removestory <id> <sentence>',
```
Comment (legnaleurc) [inline bot/__main__.py:146]: Missing `/ypc story`.

File: bot/db.py [hunk 4]
```diff
@@ -29,6 +29,13 @@ class Meme(Base):
     name = Column(String(256), nullable=False, unique=True)
     url = Column(String(65536), nullable=False)
 
+class MurmurStory(Base):
+
+    __tablename__ = 'murmurStory'
```
Comment (legnaleurc) [inline bot/db.py]: `murmur_story`

**Review by legnaleurc — CHANGES_REQUESTED** [review]
Probably need to migrate the database by myself first.

File: bot/__main__.py (hunk 3 above)
Comment (timchengx) [reply bot/__main__.py:146]: add `/ypc story <id>` after `/ypc`

File: bot/db.py (hunk 4 above)
Comment (timchengx) [reply bot/db.py]: Changed

**Comment by timchengx** [comment]
Modify code by comment.

See if it is appropriate, Thanks!

File: .gitignore [hunk 5]
```diff
@@ -1 +1,7 @@
...
+
```
Comment (legnaleurc) [inline .gitignore]: Additional newline.

File: bot/db.py [hunk 6]
```diff
@@ -29,6 +29,13 @@ class Meme(Base):
     name = Column(String(256), nullable=False, unique=True)
     url = Column(String(65536), nullable=False)
 
+class MurmurStory(Base):
+
+    __tablename__ = 'murmur_story'
+
+    id = Column (Integer, primary_key=True)
+    sentence = Column(String(65536), nullable=False)
+
```
Comment (legnaleurc) [inline bot/db.py:52]: Two empty lines between classes and functions.

**Review by legnaleurc — CHANGES_REQUESTED** [review]
I guess we need a foreign key (e.g. `murmur_id`), instead of just assume the primary key of `murmur_story` is the same to the primary key of `murmur`.

But fuck that.

File: .gitignore (hunk 5 above)
Comment (timchengx) [reply .gitignore]: Sorry, I don't follow. Should I kill line 7 or add/modify something instead?

Comment (timchengx) [reply .gitignore]: Solved, turn out my vscode automatically add additional LF for me.

File: bot/db.py (hunk 6 above)
Comment (timchengx) [reply bot/db.py:52]: Fixed.

**Comment by timchengx** [comment]
Of course, added.

**Comment by timchengx** [comment]
Ok, added.

**Review by legnaleurc — APPROVED** [review]

**Pull request merged**